domain/model/state/state.go

```diff
@@ -153,18 +206,17 @@ func Get(
 	uuid coremodel.UUID,
 ) (coremodel.Model, error) {
 	modelStmt := `
-SELECT md.name, cl.name, cr.name, mt.type, u.uuid, cc.cloud_uuid, cc.name, o.name, ccn.name, l.value
-FROM model_metadata AS md
-LEFT JOIN model_list ml ON ml.uuid = md.model_uuid
-LEFT JOIN cloud cl ON cl.uuid = md.cloud_uuid
-LEFT JOIN cloud_region cr ON cr.uuid = md.cloud_region_uuid
-LEFT JOIN cloud_credential cc ON cc.uuid = md.cloud_credential_uuid
-LEFT JOIN model_type mt ON mt.id = md.model_type_id
-LEFT JOIN user u ON u.uuid = md.owner_uuid
-LEFT JOIN user o ON o.uuid = cc.owner_uuid
-LEFT JOIN cloud ccn ON ccn.uuid = cc.cloud_uuid
-LEFT JOIN life l ON l.id = md.life_id
-WHERE md.model_uuid = ?
+SELECT name,
+       cloud_name,
+       cloud_region_name,
+       model_type_type,
+       owner_uuid,
+       cloud_credential_cloud_name,
+       cloud_credential_owner_name,
+       cloud_credential_name,
+	   life
```

Comment:
indentation looks wrong, not sure if just gh?

Reply:
Done & Done